pkg/errcode/errcode.go

```diff
@@ -643,13 +682,39 @@ A XDS RBAC policy could not be generated from the specified traffic target
 policy.
 `,
 
-	// ErrGettingLuaFilter indicates the Lua XDS HttpFilter could not be configured
 	ErrGettingLuaFilter: `
 The Lua XDS HttpFilter could not be configured.
 `,
 
-	// ErrGettingWASMFilter indicates the WASM XDS HttpFilter could not be configured
 	ErrGettingWASMFilter: `
 The WASM XDS HttpFilter could not be configured.
+`,
+
+	ErrBuildingRBACPolicyForRoute: `
+A traffic policy rule could not be configured into as a HTTP RBAC route filter.
+The route is built without an HTTP RBAC filter.
+`,
+
+	ErrUnmarshallingSDSCert: `
+A kubernetes ServiceAccount could not be unmarshalled from an SDSCert name.
+`,
+
+	ErrGettingServiceCertSecret: `
+An XDS Secret containing a new TLS certificate could not be created for the
+certificate request. The certificate request is ignored.
+`,
+
+	ErrGettingMeshService: `
+An OSM MeshService could not be configured from a SDSCert name.
+`,
+
+	ErrGettingK8sServiceAccount: `
+The SDS resource name could not be parsed and configured as an SDSCert object. The
+certificate request for the invalid SDS resource name is ignored.
+`,
+
+	ErrSDSCertMismatch: `
+The identity obtained from the SDSCert request corresponding to the mTLS root
+validation certificate does not match the identity of the proxy.
```

Comment:
```suggestion
The identity obtained from the SDS certificate request does not match the identity of the proxy.
```

Reply:
Great suggestions. Thanks for the feedback!